app/views/admin/legacy/presenters/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<h1>Presenters</h1>
```

Comment:
Not sure why the git diff picked this up as a deleted & completely new file, instead of it being moved.